tests/server/test_server.py

```diff
@@ -364,22 +394,47 @@ def test_parse_github_post(
     )
 
 
+def test_parse_github_post_delete_branch(
+    github_post_del_branch_data: bytes, github_post_del_branch_headers: Dict[str, Any]
+) -> None:
+    settings = {
+        "JD_GH_TOKEN": "GH-TOKEN",
+        "JD_GH_WEBHOOK_SECRET": "MY SECRET",
+    }
+    (request,) = parse_github_post(
+        github_post_del_branch_headers,
+        json.loads(github_post_del_branch_data),
+        github_post_del_branch_data,
+        settings,
+    )
+
+    assert request == JobsDoneRequest(
+        owner_name="ESSS",
+        repo_name="test-webhooks",
+        pusher_email="nicoddemus@gmail.com",
+        commit=None,
+        clone_url="git@github.com:ESSS/test-webhooks.git",
+        branch="fb-add-jobs-done2",
+        jobs_done_file_contents=None,
```

Comment:
Is the `jobs_done_file_contents=None` the marker for "delete jobs"?

Reply:
Yep!